### DISCUSSION TITLE
createArrowFiles has encountered an error, checking if any ArrowFiles completed

### DISCUSSION THREAD

**Discussion by lingkong966926** [GreenleafLab/ArchR#1412] [Questions / Documentation] [answered]
Hi. @rcorces ,
When I run:
ArrowFiles <- createArrowFiles(
inputFiles = inputFiles,
minFrags = 1000,
filterFrags = 1000,
filterTSS = 4
)
it shows that  createArrowFiles has encountered an error, checking if any ArrowFiles completed..
my input tsv is this.
![head](https://user-images.githubusercontent.com/93807823/165875912-f45fd255-f3b5-4f60-b5d5-55eaef9bda92.png)

**Comment by lingkong966926** [comment, marked as answer]
[Uploading ArchR-createArrows-35915839594a19-Date-2022-04-29_Time-10-14-31.log…]()

**Reply by rcorces** [reply]
your log file did not upload. fragments file needs to be position sorted, not barcode sorted

**Reply by lingkong966926** [reply]
OK,I will try it !Thanks!!!

**Reply by lingkong966926** [reply]
After I sort the fragment file,it works!!!Thanks you so much!!!
